### ISSUE TITLE
CreateContext

### ISSUE THREAD

**Issue by Tafari** [swistakm/pyimgui#122]
Having installed with the instructions on the github page (pip install imgui[full]), upon trying to run the Basic GUI drawing loop example from the documentation (https://pyimgui.readthedocs.io/en/latest/guide/first-steps.html) I get the following error:

Traceback (most recent call last):
  File "main.py", line 4, in <module>
    imgui.new_frame()
  File "imgui/core.pyx", line 1413, in imgui.core.new_frame
imgui.core.ImGuiError: ImGui assertion error (GImGui != NULL && "No current context. Did you call ImGui::CreateContext() or ImGui::SetCurrentContext()?") at imgui-cpp/imgui.cpp:3083

**Comment by swistakm** [comment]
The example was missing following snippet in order too not raise any exception:
```
  imgui.create_context()
  imgui.get_io().display_size = 100, 100
  imgui.get_io().fonts.get_tex_data_as_rgba32()
```

I have updated the docs and it should be soon available on readthedocs site.  Remember that this snippet is not enough to actually display something and you will have to use some of the built-in renderers or build one by yourself (everything should be in the docs).

**Comment by swistakm** [comment]
This has been released in [1.1.0](https://github.com/swistakm/pyimgui/releases/tag/1.1.0) and is available for download from PyPI.
